src/sasquatchbackpack/sasquatch.py

```diff
@@ -0,0 +1,127 @@
+"""Handles dispatch of backpack data to kafka."""
+
+import json
+from dataclasses import dataclass
+
+import requests
+
+from sasquatchbackpack import sources
+
+# Code yoinked from https://github.com/lsst-sqre/
+# sasquatch/blob/main/examples/RestProxyAPIExample.ipynb
+
+
+@dataclass
+class DispatcherConfig:
+    """Class containing relevant configuration information for the
+    BackpackDispatcher.
+    """
+
+    sasquatch_rest_proxy_url = (
+        "https://data-int.lsst.cloud/sasquatch-rest-proxy"
```

Comment:
When we deploy sasquatch-backpack later, we want to be able to set the `sasquatch_rest_proxy_url` from the environment. Here I would recommend:
```
 sasquatch_rest_proxy_url = os.getenv("SASQUATCH_REST_PROXY_URL", "https://data-int.lsst.cloud/sasquatch-rest-proxy")
 ```
we can now use the URL for the `data-int` environment as default. Next, we'll implement tests, and we can set the local URL as the default one.

Also, you can add docs to each configuration setting in this class.